src/ValidatorKeysTool.cpp

```diff
@@ -115,17 +143,50 @@ void createRevocation (const boost::filesystem::path& keyFile)
     std::cout << std::endl;
 }
 
-int runCommand (const std::string& command,
-    const boost::filesystem::path& keyFile)
+void signData (std::string const& data,
+    boost::filesystem::path const& keyFile)
+{
+    using namespace ripple;
+
+    if (data.empty())
+        throw std::runtime_error (
+            "Syntax error: Must specify data string to sign");
+
+    auto keys = ValidatorKeys::make_ValidatorKeys (keyFile);
+
+    if (keys.revoked())
+        std::cout << "WARNING: Validator keys have been revoked!\n\n";
+
+    std::cout << keys.sign (data) << std::endl;
+    std::cout << std::endl;
+}
+
+int runCommand (std::string const& command,
+    std::vector <std::string> const& args,
+    boost::filesystem::path const& keyFile)
 {
+    static std::map<std::string, bool> const commandArgs = {
```

Comment:
Because the value of the `map` is being compared to `args.size()`, it should be `std::vector::size_type` instead of `bool`.